code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -308,6 +308,12 @@
 	armor = list(melee = 40, bullet = 10, laser = 20, energy = 10, bomb = 30, bio = 0, rad = 0)
 	siemens_coefficient = 0.35
 
+/obj/item/clothing/suit/storage/toggle/leather_jacket/flight/legion/alt
+	name = "tcfl flight jacket"
+	desc = "A Tau Ceti Foreign Legion pilot's jacket. This is the more common, less durable variety, which typically finds itself percolating amongst all ranks of the TCFL."
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 10, rad = 0)
```

Comment:
It'd be far less snowflake to just invert the types.

Reply:
what

Reply:
I believe he means that it'd be preferable if flight/legion was the standard one, and flight/legion/alt be the armored one.

Reply:
oh. yeah, probably.